### PR TITLE
Front page and Betting page

### PR DESCRIPTION
**Pull request by ASc88**

### CODE REVIEW

File: app/app.css [hunk 1]
```diff
@@ -0,0 +1,629 @@
+/* app css stylesheet */
```
Comment (TudorStegarescu) [inline app/app.css]: please use SCSS files instead of css

File: app/components/angular-sanitize.min.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
```
Comment (TudorStegarescu) [inline app/components/angular-sanitize.min.js]: remove from branch

File: app/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  <script src="user/user.js"></script>
+
+  <script src="components/version/version.js"></script>
+  <script src="components/version/version-directive.js"></script>
+  <script src="components/version/interpolate-filter.js"></script>
+
+  <script src="../node_modules/angular-animate/angular-animate.min.js"></script>
+  <script src="../node_modules/angular-aria/angular-aria.min.js"></script>
+  <script src="../node_modules/angular-material/angular-material.min.js"></script>
+  <script src="../node_modules/angular-messages/angular-messages.min.js"></script>
```
Comment (TudorStegarescu) [inline app/index.html]: please include gulp injector instead of including each file

File: app/app.css [hunk 4]
```diff
@@ -0,0 +1,640 @@
...
+  font-size: 1.4rem;
+  line-height: 2rem;
+  letter-spacing: 0.01rem;
+  color: #212121;
+  background-color: #f5f5f5;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
+}
+#demo {
```
Comment (Graidenix) [inline app/app.css]: don't use IDs in CSS

**Pull request closed without merging**